### DISCUSSION TITLE
wiki/help-pages/flame-platform/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#644] [Wiki]
# wiki/help-pages/flame-platform/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/help-pages/flame-platform/

**Comment by PwRhGaStPY** [comment]
I'm trying to download Lucky Block for 1.20.1 from CurseForge, but the version selector lists "No Valid version found." 
I would guess this is when downloading via browser is required, but is there any way to get around that?

**Comment by goxzvt** [comment]
I'm having the same issue as the other guy

**Comment by LodusKarma** [comment]
whenever i try loading a curseforge modpack it's just a black screen
